ERROR - 2025-09-12 08:50:38 --> mysqli_sql_exception: Unknown column 'users.name' in 'field list' in F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `courses...', 0)
#1 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `courses...')
#2 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `courses...')
#3 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `courses...', Array, false)
#4 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 F:\xammp\htdocs\ITE311-GALORPOTz1\system\BaseModel.php(677): CodeIgniter\Model->doFindAll(0, 0)
#6 F:\xammp\htdocs\ITE311-GALORPOTz1\app\Models\CourseModel.php(34): CodeIgniter\BaseModel->findAll()
#7 F:\xammp\htdocs\ITE311-GALORPOTz1\app\Controllers\Home.php(23): App\Models\CourseModel->getRecentCourses(6)
#8 F:\xammp\htdocs\ITE311-GALORPOTz1\system\CodeIgniter.php(933): App\Controllers\Home->index()
#9 F:\xammp\htdocs\ITE311-GALORPOTz1\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#10 F:\xammp\htdocs\ITE311-GALORPOTz1\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#12 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 F:\xammp\htdocs\ITE311-GALORPOTz1\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-09-12 08:50:38 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'users.name' in 'field list'
[Method: GET, Route: /]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `courses`.*, `users`.`name` as `instructor_name`
FROM `courses`
JOIN `users` ON `users`.`id` = `courses`.`instructor_id`
ORDER BY `courses`.`created_at` DESC
 LIMIT 6', [], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(677): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Models\CourseModel.php(34): CodeIgniter\BaseModel->findAll()
 5 APPPATH\Controllers\Home.php(23): App\Models\CourseModel->getRecentCourses(6)
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 08:50:38 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'users.name' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `courses`.*, `users`.`name` as `instructor_name`
FROM `courses`
JOIN `users` ON `users`.`id` = `courses`.`instructor_id`
ORDER BY `courses`.`created_at` DESC
 LIMIT 6')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `courses`.*, `users`.`name` as `instructor_name`
FROM `courses`
JOIN `users` ON `users`.`id` = `courses`.`instructor_id`
ORDER BY `courses`.`created_at` DESC
 LIMIT 6')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `courses`.*, `users`.`name` as `instructor_name`
FROM `courses`
JOIN `users` ON `users`.`id` = `courses`.`instructor_id`
ORDER BY `courses`.`created_at` DESC
 LIMIT 6', [], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(677): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Models\CourseModel.php(34): CodeIgniter\BaseModel->findAll()
 7 APPPATH\Controllers\Home.php(23): App\Models\CourseModel->getRecentCourses(6)
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 08:50:38 --> [Caused by] mysqli_sql_exception: Unknown column 'users.name' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `courses`.*, `users`.`name` as `instructor_name`
FROM `courses`
JOIN `users` ON `users`.`id` = `courses`.`instructor_id`
ORDER BY `courses`.`created_at` DESC
 LIMIT 6', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `courses`.*, `users`.`name` as `instructor_name`
FROM `courses`
JOIN `users` ON `users`.`id` = `courses`.`instructor_id`
ORDER BY `courses`.`created_at` DESC
 LIMIT 6')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `courses`.*, `users`.`name` as `instructor_name`
FROM `courses`
JOIN `users` ON `users`.`id` = `courses`.`instructor_id`
ORDER BY `courses`.`created_at` DESC
 LIMIT 6')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `courses`.*, `users`.`name` as `instructor_name`
FROM `courses`
JOIN `users` ON `users`.`id` = `courses`.`instructor_id`
ORDER BY `courses`.`created_at` DESC
 LIMIT 6', [], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(677): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Models\CourseModel.php(34): CodeIgniter\BaseModel->findAll()
 8 APPPATH\Controllers\Home.php(23): App\Models\CourseModel->getRecentCourses(6)
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-09-12 08:51:57 --> mysqli_sql_exception: Table 'lms_galorpot.users' doesn't exist in F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', NULL, false)
#4 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
#5 F:\xammp\htdocs\ITE311-GALORPOTz1\app\Controllers\Home.php(18): CodeIgniter\Model->__call('countAll', Array)
#6 F:\xammp\htdocs\ITE311-GALORPOTz1\system\CodeIgniter.php(933): App\Controllers\Home->index()
#7 F:\xammp\htdocs\ITE311-GALORPOTz1\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 F:\xammp\htdocs\ITE311-GALORPOTz1\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#10 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 F:\xammp\htdocs\ITE311-GALORPOTz1\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-09-12 08:51:57 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'lms_galorpot.users' doesn't exist
[Method: GET, Route: /]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `users`', null, false)
 2 SYSTEMPATH\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 3 APPPATH\Controllers\Home.php(18): CodeIgniter\Model->__call('countAll', [])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 08:51:57 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'lms_galorpot.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows` FROM `users`')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows` FROM `users`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `users`', null, false)
 4 SYSTEMPATH\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 5 APPPATH\Controllers\Home.php(18): CodeIgniter\Model->__call('countAll', [])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 08:51:57 --> [Caused by] mysqli_sql_exception: Table 'lms_galorpot.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*) AS `numrows` FROM `users`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows` FROM `users`')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows` FROM `users`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `users`', null, false)
 5 SYSTEMPATH\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 6 APPPATH\Controllers\Home.php(18): CodeIgniter\Model->__call('countAll', [])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-09-12 08:51:58 --> mysqli_sql_exception: Table 'lms_galorpot.users' doesn't exist in F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', NULL, false)
#4 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
#5 F:\xammp\htdocs\ITE311-GALORPOTz1\app\Controllers\Home.php(18): CodeIgniter\Model->__call('countAll', Array)
#6 F:\xammp\htdocs\ITE311-GALORPOTz1\system\CodeIgniter.php(933): App\Controllers\Home->index()
#7 F:\xammp\htdocs\ITE311-GALORPOTz1\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 F:\xammp\htdocs\ITE311-GALORPOTz1\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#10 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 F:\xammp\htdocs\ITE311-GALORPOTz1\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-09-12 08:51:58 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'lms_galorpot.users' doesn't exist
[Method: GET, Route: /]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `users`', null, false)
 2 SYSTEMPATH\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 3 APPPATH\Controllers\Home.php(18): CodeIgniter\Model->__call('countAll', [])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 08:51:58 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'lms_galorpot.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows` FROM `users`')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows` FROM `users`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `users`', null, false)
 4 SYSTEMPATH\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 5 APPPATH\Controllers\Home.php(18): CodeIgniter\Model->__call('countAll', [])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 08:51:58 --> [Caused by] mysqli_sql_exception: Table 'lms_galorpot.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*) AS `numrows` FROM `users`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows` FROM `users`')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows` FROM `users`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `users`', null, false)
 5 SYSTEMPATH\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 6 APPPATH\Controllers\Home.php(18): CodeIgniter\Model->__call('countAll', [])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-09-12 08:51:58 --> mysqli_sql_exception: Table 'lms_galorpot.users' doesn't exist in F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', NULL, false)
#4 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
#5 F:\xammp\htdocs\ITE311-GALORPOTz1\app\Controllers\Home.php(18): CodeIgniter\Model->__call('countAll', Array)
#6 F:\xammp\htdocs\ITE311-GALORPOTz1\system\CodeIgniter.php(933): App\Controllers\Home->index()
#7 F:\xammp\htdocs\ITE311-GALORPOTz1\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 F:\xammp\htdocs\ITE311-GALORPOTz1\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#10 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 F:\xammp\htdocs\ITE311-GALORPOTz1\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-09-12 08:51:58 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'lms_galorpot.users' doesn't exist
[Method: GET, Route: /]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `users`', null, false)
 2 SYSTEMPATH\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 3 APPPATH\Controllers\Home.php(18): CodeIgniter\Model->__call('countAll', [])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 08:51:58 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'lms_galorpot.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows` FROM `users`')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows` FROM `users`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `users`', null, false)
 4 SYSTEMPATH\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 5 APPPATH\Controllers\Home.php(18): CodeIgniter\Model->__call('countAll', [])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 08:51:58 --> [Caused by] mysqli_sql_exception: Table 'lms_galorpot.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*) AS `numrows` FROM `users`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows` FROM `users`')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows` FROM `users`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `users`', null, false)
 5 SYSTEMPATH\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 6 APPPATH\Controllers\Home.php(18): CodeIgniter\Model->__call('countAll', [])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-09-12 08:51:58 --> mysqli_sql_exception: Table 'lms_galorpot.users' doesn't exist in F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', NULL, false)
#4 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
#5 F:\xammp\htdocs\ITE311-GALORPOTz1\app\Controllers\Home.php(18): CodeIgniter\Model->__call('countAll', Array)
#6 F:\xammp\htdocs\ITE311-GALORPOTz1\system\CodeIgniter.php(933): App\Controllers\Home->index()
#7 F:\xammp\htdocs\ITE311-GALORPOTz1\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 F:\xammp\htdocs\ITE311-GALORPOTz1\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#10 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 F:\xammp\htdocs\ITE311-GALORPOTz1\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-09-12 08:51:58 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'lms_galorpot.users' doesn't exist
[Method: GET, Route: /]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `users`', null, false)
 2 SYSTEMPATH\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 3 APPPATH\Controllers\Home.php(18): CodeIgniter\Model->__call('countAll', [])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 08:51:58 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'lms_galorpot.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows` FROM `users`')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows` FROM `users`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `users`', null, false)
 4 SYSTEMPATH\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 5 APPPATH\Controllers\Home.php(18): CodeIgniter\Model->__call('countAll', [])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 08:51:58 --> [Caused by] mysqli_sql_exception: Table 'lms_galorpot.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*) AS `numrows` FROM `users`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows` FROM `users`')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows` FROM `users`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `users`', null, false)
 5 SYSTEMPATH\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 6 APPPATH\Controllers\Home.php(18): CodeIgniter\Model->__call('countAll', [])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-09-12 08:51:58 --> mysqli_sql_exception: Table 'lms_galorpot.users' doesn't exist in F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', NULL, false)
#4 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
#5 F:\xammp\htdocs\ITE311-GALORPOTz1\app\Controllers\Home.php(18): CodeIgniter\Model->__call('countAll', Array)
#6 F:\xammp\htdocs\ITE311-GALORPOTz1\system\CodeIgniter.php(933): App\Controllers\Home->index()
#7 F:\xammp\htdocs\ITE311-GALORPOTz1\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 F:\xammp\htdocs\ITE311-GALORPOTz1\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#10 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 F:\xammp\htdocs\ITE311-GALORPOTz1\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-09-12 08:51:58 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'lms_galorpot.users' doesn't exist
[Method: GET, Route: /]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `users`', null, false)
 2 SYSTEMPATH\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 3 APPPATH\Controllers\Home.php(18): CodeIgniter\Model->__call('countAll', [])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 08:51:58 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'lms_galorpot.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows` FROM `users`')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows` FROM `users`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `users`', null, false)
 4 SYSTEMPATH\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 5 APPPATH\Controllers\Home.php(18): CodeIgniter\Model->__call('countAll', [])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 08:51:58 --> [Caused by] mysqli_sql_exception: Table 'lms_galorpot.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*) AS `numrows` FROM `users`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows` FROM `users`')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows` FROM `users`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `users`', null, false)
 5 SYSTEMPATH\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 6 APPPATH\Controllers\Home.php(18): CodeIgniter\Model->__call('countAll', [])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-09-12 08:51:58 --> mysqli_sql_exception: Table 'lms_galorpot.users' doesn't exist in F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', NULL, false)
#4 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
#5 F:\xammp\htdocs\ITE311-GALORPOTz1\app\Controllers\Home.php(18): CodeIgniter\Model->__call('countAll', Array)
#6 F:\xammp\htdocs\ITE311-GALORPOTz1\system\CodeIgniter.php(933): App\Controllers\Home->index()
#7 F:\xammp\htdocs\ITE311-GALORPOTz1\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 F:\xammp\htdocs\ITE311-GALORPOTz1\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#10 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 F:\xammp\htdocs\ITE311-GALORPOTz1\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-09-12 08:51:58 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'lms_galorpot.users' doesn't exist
[Method: GET, Route: /]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `users`', null, false)
 2 SYSTEMPATH\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 3 APPPATH\Controllers\Home.php(18): CodeIgniter\Model->__call('countAll', [])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 08:51:58 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'lms_galorpot.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows` FROM `users`')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows` FROM `users`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `users`', null, false)
 4 SYSTEMPATH\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 5 APPPATH\Controllers\Home.php(18): CodeIgniter\Model->__call('countAll', [])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 08:51:58 --> [Caused by] mysqli_sql_exception: Table 'lms_galorpot.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*) AS `numrows` FROM `users`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows` FROM `users`')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows` FROM `users`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `users`', null, false)
 5 SYSTEMPATH\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 6 APPPATH\Controllers\Home.php(18): CodeIgniter\Model->__call('countAll', [])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-09-12 08:51:58 --> mysqli_sql_exception: Table 'lms_galorpot.users' doesn't exist in F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', NULL, false)
#4 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
#5 F:\xammp\htdocs\ITE311-GALORPOTz1\app\Controllers\Home.php(18): CodeIgniter\Model->__call('countAll', Array)
#6 F:\xammp\htdocs\ITE311-GALORPOTz1\system\CodeIgniter.php(933): App\Controllers\Home->index()
#7 F:\xammp\htdocs\ITE311-GALORPOTz1\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 F:\xammp\htdocs\ITE311-GALORPOTz1\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#10 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 F:\xammp\htdocs\ITE311-GALORPOTz1\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-09-12 08:51:58 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'lms_galorpot.users' doesn't exist
[Method: GET, Route: /]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `users`', null, false)
 2 SYSTEMPATH\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 3 APPPATH\Controllers\Home.php(18): CodeIgniter\Model->__call('countAll', [])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 08:51:58 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'lms_galorpot.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows` FROM `users`')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows` FROM `users`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `users`', null, false)
 4 SYSTEMPATH\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 5 APPPATH\Controllers\Home.php(18): CodeIgniter\Model->__call('countAll', [])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 08:51:58 --> [Caused by] mysqli_sql_exception: Table 'lms_galorpot.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*) AS `numrows` FROM `users`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows` FROM `users`')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows` FROM `users`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `users`', null, false)
 5 SYSTEMPATH\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 6 APPPATH\Controllers\Home.php(18): CodeIgniter\Model->__call('countAll', [])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-09-12 08:52:14 --> mysqli_sql_exception: Table 'lms_galorpot.users' doesn't exist in F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', NULL, false)
#4 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
#5 F:\xammp\htdocs\ITE311-GALORPOTz1\app\Controllers\Home.php(18): CodeIgniter\Model->__call('countAll', Array)
#6 F:\xammp\htdocs\ITE311-GALORPOTz1\system\CodeIgniter.php(933): App\Controllers\Home->index()
#7 F:\xammp\htdocs\ITE311-GALORPOTz1\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 F:\xammp\htdocs\ITE311-GALORPOTz1\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#10 F:\xammp\htdocs\ITE311-GALORPOTz1\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 F:\xammp\htdocs\ITE311-GALORPOTz1\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-09-12 08:52:14 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'lms_galorpot.users' doesn't exist
[Method: GET, Route: /]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `users`', null, false)
 2 SYSTEMPATH\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 3 APPPATH\Controllers\Home.php(18): CodeIgniter\Model->__call('countAll', [])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 08:52:14 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'lms_galorpot.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows` FROM `users`')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows` FROM `users`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `users`', null, false)
 4 SYSTEMPATH\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 5 APPPATH\Controllers\Home.php(18): CodeIgniter\Model->__call('countAll', [])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 08:52:14 --> [Caused by] mysqli_sql_exception: Table 'lms_galorpot.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*) AS `numrows` FROM `users`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows` FROM `users`')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows` FROM `users`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `users`', null, false)
 5 SYSTEMPATH\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 6 APPPATH\Controllers\Home.php(18): CodeIgniter\Model->__call('countAll', [])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 09:05:26 --> ParseError: syntax error, unexpected token ":"
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(821): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 09:05:26 --> ParseError: syntax error, unexpected token ":"
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(821): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 09:05:55 --> ParseError: syntax error, unexpected token ":"
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(821): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 09:06:00 --> ParseError: syntax error, unexpected token ":"
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(821): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 09:06:01 --> ParseError: syntax error, unexpected token ":"
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(821): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 09:06:01 --> ParseError: syntax error, unexpected token ":"
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(821): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 09:06:01 --> ParseError: syntax error, unexpected token ":"
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(821): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 09:06:01 --> ParseError: syntax error, unexpected token ":"
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(821): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 09:06:04 --> ParseError: syntax error, unexpected token ":"
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(821): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 09:06:15 --> ParseError: syntax error, unexpected token ":"
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(821): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 09:06:16 --> ParseError: syntax error, unexpected token ":"
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(821): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 09:06:16 --> ParseError: syntax error, unexpected token ":"
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(821): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 09:07:23 --> ParseError: syntax error, unexpected token ":"
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(821): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 09:07:24 --> ParseError: syntax error, unexpected token ":"
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(821): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 09:07:24 --> ParseError: syntax error, unexpected token ":"
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(821): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 09:07:38 --> ParseError: syntax error, unexpected token ":"
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(821): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 09:07:39 --> ParseError: syntax error, unexpected token ":"
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(821): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 09:07:39 --> ParseError: syntax error, unexpected token ":"
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(821): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 09:07:39 --> ParseError: syntax error, unexpected token ":"
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(821): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 09:07:39 --> ParseError: syntax error, unexpected token ":"
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(821): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-12 09:07:46 --> ParseError: syntax error, unexpected token ":"
[Method: CLI, Route: migrate]
in APPPATH\Config\Routes.php on line 10.
 1 SYSTEMPATH\CLI\Console.php(41): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\Boot.php(388): CodeIgniter\CLI\Console->run()
 3 SYSTEMPATH\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
 4 ROOTPATH\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
